workspaces/redhat-argocd/plugins/argocd-backend/package.json

```diff
@@ -0,0 +1,73 @@
+{
+  "name": "@backstage-community/plugin-redhat-argocd-backend",
+  "version": "0.1.0",
+  "main": "src/index.ts",
+  "types": "src/index.ts",
+  "license": "Apache-2.0",
+  "publishConfig": {
+    "access": "public",
+    "main": "dist/index.cjs.js",
+    "types": "dist/index.d.ts"
+  },
+  "backstage": {
+    "role": "backend-plugin",
+    "supported-versions": "1.35.0",
+    "pluginId": "redhat-argocd-backend",
+    "pluginPackage": "@backstage-community/plugin-redhat-argocd-backend",
+    "pluginPackages": [
+      "@backstage-community/plugin-redhat-argocd-backend"
+    ]
+  },
+  "scripts": {
+    "start": "backstage-cli package start",
+    "build": "backstage-cli package build",
+    "lint": "backstage-cli package lint",
+    "test": "backstage-cli package test",
+    "clean": "backstage-cli package clean",
+    "prepack": "backstage-cli package prepack",
+    "postpack": "backstage-cli package postpack"
+  },
+  "dependencies": {
+    "@backstage-community/plugin-redhat-argocd-common": "workspace:^",
+    "@backstage/backend-defaults": "^0.7.0",
+    "@backstage/backend-plugin-api": "^1.1.1",
+    "@backstage/catalog-client": "^1.9.1",
+    "@backstage/config": "^1.3.2",
+    "@backstage/errors": "^1.2.7",
+    "@backstage/plugin-catalog-node": "^1.15.1",
+    "@backstage/plugin-permission-common": "^0.8.4",
+    "@backstage/plugin-permission-node": "^0.8.7",
+    "@kubernetes/client-node": "^0.22.1",
+    "express": "^4.17.1",
+    "express-promise-router": "^4.1.0",
+    "undici": "^7.3.0",
+    "zod": "^3.22.4"
+  },
+  "devDependencies": {
+    "@backstage/backend-test-utils": "^1.2.1",
+    "@backstage/cli": "^0.29.5",
+    "@types/express": "^4.17.6",
+    "@types/supertest": "^2.0.12",
+    "supertest": "^6.2.4"
+  },
+  "files": [
```

Comment:
You're missing the `config.d.ts` as well as the schema as per the docs: https://backstage.io/docs/conf/defining

Reply:
Nice catch, thank you! Does this look good? -> https://github.com/backstage/community-plugins/pull/2837/commits/d7a242b92b506a7c373c7a51c8be7248b45afb4d